bucket/presentmon.json

```diff
@@ -0,0 +1,35 @@
+{
+    "version": "1.7.0",
+    "description": "Tool for collection and processing of ETW events related to frame presentation on Windows.",
+    "homepage": "https://github.com/GameTechDev/PresentMon",
+    "license": "MIT",
+    "pre_install": "Rename-Item -Path \"$DIR\\PresentMon-*-x*.exe\" -NewName \"$DIR\\PresentMon.exe\"",
+    "architecture":{
+        "64bit":{
+            "url": "https://github.com/GameTechDev/PresentMon/releases/download/v1.7.0/PresentMon-1.7.0-x64.exe",
+            "hash": "E1C475B66AD87ED4ADD950F0F29B31E53D6093CDB8182D0FDD1B6B7B690E14F1"
+        },
+        "32bit":{
+            "url": "https://github.com/GameTechDev/PresentMon/releases/download/v1.7.0/PresentMon-1.7.0-x86.exe",
+            "hash": "163999393086B349C09FDBAC74FB1C151564FFB07FF094F4E4ADCE22FD5D8098"
+        }
+    },
+    "bin": [
+        "PresentMon.exe",
+        [
+            "PresentMon.exe",
+            "pm"
```

Comment:
Any documentation on this particular alias?

Reply:
no

Reply:
Then why add it?

Reply:
to be frank I like it and tried to snug it into the Main bucket

Reply:
The next release of Scoop will add the ability to add custom aliases, you can use that then